cmd/delete.go

```diff
@@ -0,0 +1,27 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {}
+
+var deleteCmd = &cobra.Command{
+	Use:   "delete [url]",
```

Comment:
should mention that url means Short URL instead of full URL, as this fails if users input full URL